website/content/en/status/report-2024-10-2024-12/azure.adoc

```diff
@@ -0,0 +1,40 @@
+=== FreeBSD on Microsoft HyperV and Azure
+
+Links: +
+link:https://wiki.freebsd.org/MicrosoftAzure[Microsoft Azure article on FreeBSD wiki] URL: link:https://wiki.freebsd.org/MicrosoftAzure[] +
+link:https://wiki.freebsd.org/HyperV[Microsoft HyperV article on FreeBSD wiki] URL: link:https://wiki.freebsd.org/HyperV[]
+
+Contact: Microsoft FreeBSD Integration Services Team <bsdic@microsoft.com> +
+Contact: link:https://lists.freebsd.org/subscription/freebsd-cloud[freebsd-cloud Mailing List] +
+Contact: The FreeBSD Azure Release Engineering Team <releng-azure@FreeBSD.org> +
+Contact: Wei Hu <whu@FreeBSD.org>, <weh@microsoft.com> +
+Contact: Souradeep Chakrabarti <schakrabarti@microsoft.com> +
+Contact: Colin Su <yuas@microsoft.com> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org>
+
+In this quarter, we have published the link:https://azuremarketplace.microsoft.com/marketplace/apps/thefreebsdfoundation.freebsd-14_2[14.2-RELEASE on Azure Marketplace].
+
+Colin Su has presented at the FreeBSD 2024 Fall Sumit about Azure DevOps Pipeline.
+
+Souradeep Chakrabarti from Microsoft has added a feature to use hypercalls for TLB shootdown on Hyper-V and Azure.
+
+Wei Hu root-caused an issue on missing CDROM device when booting FreeBSD on the latest Azure v6 VM SKU.
+V6 type only offers NVMe disks to guest OS.
+He also continues bug fixing for FreeBSD MANA NIC device.
+
+Work in progress tasks:
+
+* Automating the image publishing process and merging to [.filename]#src/release/#. (Li-Wen Hsu)
+* Colin Su is testing adding FreeBSD support in link:https://azure.microsoft.com/products/devops/pipelines/[Azure Pipelines]
+** https://github.com/microsoft/azure-pipelines-agent/pull/3266[]
```

Comment:
Please always use the `link:` macro explicitly. I know that it works even without it, but we need it for some reaon (I think it is related to translations).
```suggestion
** link:https://github.com/microsoft/azure-pipelines-agent/pull/3266[]
```

I would also add some descriptive text within the square parenthesis `[ ]` if possible.